packages/vuetify/src/components/VCalendar/VCalendar.ts

```diff
@@ -293,11 +293,11 @@ export default CalendarWithEvents.extend({
     },
     getCategoryList (categories: CalendarCategory[]): CalendarCategory[] {
       if (!this.noEvents) {
-        const categoryMap = categories.reduce((map, category, index) => {
+        const categoryMap: any = categories.reduce((map: any, category, index) => {
```

Comment:
This uses `categories[0]` as `map`. If an object doesn't have `hasOwnProperty` you call `Object.prototype.hasOwnProperty.call(obj, prop)` instead.

Reply:
I'm not the one who wrote that code. It's been in there since Vcalendar, i'd rather not mess with it, I just needed to remove the non inherited portion.

Reply:
I'm saying that's what your change does.